apps/pl/webpack.pl.prod.js

```diff
@@ -22,6 +21,13 @@ const pl = {
     new webpack.DefinePlugin({
       BUILD_TARGET: JSON.stringify('pl'),
     }),
+    new webpack.ProvidePlugin({
```

Comment:
I don't think this needs to be here since it's provided all the way up in `webpack.shared.dev.js`.